wgpu-core/src/track/stateless.rs

```diff
@@ -128,11 +133,10 @@ impl<A: hub::HalApi, T: hub::Resource, Id: TypedId> StatelessTracker<A, T, Id> {
         self.tracker_assert_in_bounds(index);
 
         unsafe {
-            self.metadata
-                .insert(index, epoch, item.life_guard().add_ref());
+            self.metadata.insert(index, epoch, resource.clone());
```

Comment:
I profiled bunnymark, and this arc.clone() appears to be taking ~40% of the CPU time with 10k bunnies (and ~50% with 125k bunnies).